build/requirements.in

```diff
@@ -20,10 +20,13 @@ jaxlib==0.6.1
 
 # The with-cuda extra also includes NVIDIA's pip packages.
 jax-cuda12-plugin[with-cuda]==0.6.1 ; sys_platform == "linux"
+jax-cuda13-plugin ; sys_platform == "does_not_exist"
```

Comment:
If the effect of the sys_platform guard is to disable this, why add it all? Maybe just comment it out or something?